### DISCUSSION TITLE
What is a thing? Is a thing a thing? Is nothing a thing?

### DISCUSSION THREAD

**Discussion by floomm** [riesentoaster/geiman01#14] [Thingification] [answered]
Personally, I feel everything is a thing. Now, think about things.

**Comment by Geissemanu** [comment, marked as answer]
Thing -> Something i.e. "thats quite something"
Maybething -> maybe a thing i.e. "maybe this could become a thing"
Nothing -> Not a thing i.e. "i'm fine its nothing"

**Comment by riesentoaster** [comment]
I feel like maybethings are things (although they might not actually be things), where we couldn't decide yet, whether or not something is a thing.

**Reply by Geissemanu** [reply]
this would implicate, that the whole discusion is a maybething?

**Reply by riesentoaster** [reply]
I'm not sure whether a meta-attribute of a maybething inherits the thingification-category of the parent.
